Currency-React-Web-App/Program.cs

```diff
@@ -1,32 +1,52 @@
-﻿using System.Net.Http.Headers;
+﻿using System.Collections.Specialized;
+using System.Net.Http.Headers;
 using Currency_React_Web_App.Clients;
-using Currency_React_Web_App.Controllers;
 using Currency_React_Web_App.Services;
 using MongoDB.Driver;
+using Quartz;
+using Quartz.Impl;
+using System.ServiceProcess;
 
 namespace Currency_React_Web_App;
 
 public class Program
 {
+    public Program ()
+    {
+        
+    }
+
+    private static IScheduler _quartzScheduler = null!;
+    
     public static void Main(string[] args)
     {
         var builder = WebApplication.CreateBuilder(args);
-
+        var services = builder.Services;
+        
         string currentCurrencyUrl = builder.Configuration.GetValue<string>("urls:BaseRequestUrl");
         string mongoClientUri = builder.Configuration.GetValue<string>("urls:MongoDBUri");
         
-        builder.Services.AddHttpClient<ICurrencyClient, CurrencyClient>(client =>
+        services.AddHttpClient<ICurrencyClient, CurrencyClient>(client =>
         {
             client.BaseAddress = new Uri(currentCurrencyUrl);
             client.DefaultRequestHeaders.Accept.Clear();
             client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
         });
-        builder.Services.AddSingleton<MongoClientBase, MongoClient>(_ => new MongoClient(mongoClientUri));
-        builder.Services.AddSingleton<ICurrentDateCurrencyService, CurrentDateCurrencyService>();
-        builder.Services.AddSingleton<IFetchDataService, FetchDataService>();
-        builder.Services.AddSingleton<ICurrentDateCurrencyCache, CurrentDateCurrencyCache>();
-        builder.Services.AddMemoryCache();
-        builder.Services.AddControllers();
+        
+        services.AddSingleton<MongoClientBase, MongoClient>(_ => new MongoClient(mongoClientUri));
+        
+        services.AddSingleton<ICurrentDateCurrencyService, CurrentDateCurrencyService>();
+        
+        services.AddSingleton<IFetchDataService, FetchDataService>();
+        
+        services.AddSingleton<ICurrentDateCurrencyCache, CurrentDateCurrencyCache>();
+        services.AddMemoryCache();
+        var a = QuartzConfig();
```

Comment:
It seems to not correct for me (how quarts is setup here).

Try to follow documentation link:
https://www.quartz-scheduler.net/documentation/quartz-3.x/packages/microsoft-di-integration.html#di-aware-job-factories